requirements/samples.txt

```diff
@@ -1 +1,2 @@
-scipy~=1.10
\ No newline at end of file
+scipy~=1.10
```

Comment:
@abramov-oleg 
Please change the `-samples` dependency to `compose build` with building a local image based on the `-base`. We are going to get rid of samples in the future.

Reply:
Removed dependency on `samples` image.